app/views/admin/users/show/flags/_index.html.erb

```diff
@@ -1,43 +1,46 @@
-<h2 class="crayons-subtitle-1">Flags received (<%= @related_vomit_reactions.count %>)</h2>
+<%= javascript_packs_with_chunks_tag "admin/shared/flagReactionItemDropdowmButton", defer: true %>
+<% user_vomit_reactions = @related_vomit_reactions.select { |flag| flag.reactable_type == "User" } %>
+<% score = 0 %>
+<% countable_flags = 0 %>
+<% if user_vomit_reactions.present? %>
+  <% user_vomit_reactions.each do |flag| %>
+    <% score += flag.points %>
+      <% if flag.status != 'invalid' %>
+        <% countable_flags += 1 %>
+      <% end %>
+  <% end %>
+<% end %>
```

Comment:
Is this logic not possible to add to a method in the controller action, or is there a reason that we add it in the template?

Reply:
Shifted this logic to controller file.